macro/run_primary_vertexer_ITS.C

```diff
@@ -16,12 +15,13 @@
 #include "ITSBase/GeometryTGeo.h"
 #include "ITStracking/IOUtils.h"
 #include "ITStracking/Vertexer.h"
+// #include "ITStrackingCUDA/VertexerTraitsGPU.h"
 
-#if defined(__VERTEXER_ITS_GPU)
+#define GPUCA_O2_LIB // Temporary workaround (Aug 13, 2019)
```

Comment:
This can be removed once #2452 is merged (which will be the case once MacOS CI is fixed), so I'd remove it here before merging then.

Reply:
Ok, let's break tests for the moment. This is an improvement.